runtimes/php/php-apps.md

```diff
@@ -333,14 +333,17 @@ how to enable them.
 
 If you have a request about modules, feel free to contact our support at <support@clever-cloud.com>.
 
+Extensions enabled by default, add `ENABLE_<extension_name>`: `false` in your environment variables to disable them:
+`couchbase`, `imagick`, `memcached`, `memcache`, `mongodb`, `opcache`, `redis`, `solr`, `ssh2`.
```

Comment:
maybe add a note saying that for now memcache and memcached are not available for php7?